updates/pgbouncer_fdw--0.4--0.5.sql

```diff
@@ -0,0 +1,216 @@
+CREATE TEMP TABLE pgbouncer_fdw_preserve_privs_temp (statement text);
```

Comment:
I assume this runs implicitly in a transaction?

Reply:
All EXENSION commands run in an implicit transaction. I aslo explicitly drop the temp table at the end too